### PR TITLE
Removed complete_script method from TreeSink trait

### LINKED ISSUE #276: Remove complete_script method from TreeSink trait
This method is no longer used in html5ever, and xml5ever could be modified to never depend on this method at all.

### PR DESCRIPTION
**Pull request by rozuur**
Bumped minor crate versions
fixes #276

### CODE REVIEW

**Comment by rozuur** [comment]
@cynicaldevil Please approve 😄

File: xml5ever/src/tree_builder/mod.rs [hunk 1]
```diff
@@ -692,7 +685,6 @@ impl<Handle, Sink> XmlTreeBuilder<Handle, Sink>
                     };
                     if tag.name.local == local_name!("script") {
                         self.insert_tag(tag.clone());
-                        self.complete_script();
```
Comment (cynicaldevil) [inline xml5ever/src/tree_builder/mod.rs]: You need to return early when you encounter a script end tag. Same goes for the other `complete_script` call below.

Comment (rozuur) [reply xml5ever/src/tree_builder/mod.rs]: Will check next week how it can be implemented.

**Comment by cynicaldevil** [comment]
@rozuur Any updates?

**Comment by rozuur** [comment]
I am currently not capable of working on this. Sorry to close this PR.

**Pull request closed without merging**